crates/relayer/src/worker/packet.rs

```diff
@@ -92,6 +104,48 @@ pub fn spawn_packet_cmd_worker<ChainA: ChainHandle, ChainB: ChainHandle>(
     })
 }
 
+pub fn spawn_incentivized_packet_cmd_worker<ChainA: ChainHandle, ChainB: ChainHandle>(
+    cmd_rx: Receiver<WorkerCmd>,
+    // Mutex is used to prevent race condition between the packet workers
+    link: Arc<Mutex<Link<ChainA, ChainB>>>,
+    path: Packet,
+    fee_filter: FeesFilters,
+) -> TaskHandle {
+    let span = {
+        let relay_path = &link.lock().unwrap().a_to_b;
+        error_span!(
+            "worker.incentivized.packet.cmd",
+            src_chain = %relay_path.src_chain().id(),
+            src_port = %relay_path.src_port_id(),
+            src_channel = %relay_path.src_channel_id(),
+            dst_chain = %relay_path.dst_chain().id(),
+        )
+    };
+
+    // This Cache will store the IncentivizedPacket observed. They will then be used in order
+    // to verify if a SendPacket event is incentivized.
+    let incentivized_recv_cache: RwArc<Cache<Sequence, IncentivizedPacket>> = RwArc::new_lock(
+        moka::sync::Cache::builder()
+            .time_to_live(Duration::from_secs(10 * 60))
+            .max_capacity(1000)
```

Comment:
Let's extract those into constants. I think we can also increase the TTL to 1min or something like that, what do you think?